setup.py

```diff
@@ -59,6 +59,6 @@ def read(fname):
     install_requires=['sagemaker-containers==2.1', 'retrying==1.3.3'],
     extras_require={
         'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist', 'mock', 'sagemaker',
-                 'requests==2.18.4', 'docker-compose']
+                 'requests==2.18.4', 'docker-compose', 'mxnet']
```

Comment:
does this get installed when we pip install sagemaker-mxnet-container? Is going to interfere with the mxnet binary we installed directly in the dockerfile?

Reply:
only gets installed for tests, though yeah, freezing the version is probably a good idea